.travis.yml

```diff
@@ -43,19 +43,19 @@ before_script:
 - ./ci/build_image.py $TEST_IMAGE --provider=$DOCKER_PROVIDER --quiet
 script:
 - flake8 $(git ls-files '*.py')
-- COVERAGE_PROCESS_START=${TRAVIS_BUILD_DIR}/.coveragerc ci/test.py -vb PyrexImage_${DOCKER_PROVIDER}_$(echo $TEST_IMAGE | sed 's/\W/_/g')
+- COVERAGE_PROCESS_START=${TRAVIS_BUILD_DIR}/.coveragerc $SH -c "ci/test.py -vb PyrexImage_${DOCKER_PROVIDER}_$(echo $TEST_IMAGE | sed 's/\W/_/g')"
 after_success:
 - coverage3 combine
 - coveralls
 deploy:
 - provider: script
-  skip_cleanup: true
+  cleanup: false
```

Comment:
Is this just a refresh to some new syntax available in the TravisCI config file?

Reply:
Correct